src/status_im/contexts/profile/settings/view.cljs

```diff
@@ -78,7 +77,7 @@
         :customization-color customization-color
         :label               (i18n/label :t/jump-to)}}
       {:position :absolute
-       :bottom   jump-to.constants/floating-shell-button-height}]]))
+       :bottom   20}]]))
```

Comment:
Thank you @alwx for fixing this issue.

Please don't hardcode 20, I think we can use `(:bottom insets) here. wdyt?

Reply:
Also, if possible Please also make same change in `status-im.contexts.profile.contact.view`. (It seems they are copy pasted, and using bottom wrong)

Reply:
`(:bottom insets)` has a different value, on Androids it can be zero, on iPhones with Face ID it's 34. 
I would say it needs to be *at least* 20, and on iPhones it should be even higher just because the bottom part of the screen is already occupied with a pill that's used for multitasking / minimizing the app.

Reply:
> I would say it needs to be at least 20, and on iPhones it should be even higher just because the bottom part of the screen is already occupied with a pill that's used for multitasking / minimizing the app.

Yes, that's what bottom insets will provide. Height of that multitasking area.

![image](https://github.com/status-im/status-mobile/assets/17097240/66b55fe1-12f9-4630-9e95-2f71bfa71510)

> on Androids it can be zero, 

what are issues with that? Even if is zero, it is position of floating-shell-button container as same as figma.
And jump-to position will not be zero(because container already position jump-to close to the top).

If we keep this bottom padding also for android by hard coding where we don't have multitasking button then we are unnecessarily increasing this space. Similar to  screenshot shared in issue.

![image](https://github.com/status-im/status-mobile/assets/17097240/10885cba-879a-435d-8269-c370cdfa01cd)

Reply:
I don't get what you're talking about — the bottom inset on latest iPhones is 34 and not 21.

> Even if is zero, it is position of floating-shell-button container as same as figma.

I cannot really parse this sentence, sorry. Could you rephrase it to make it clear what you mean?

Reply:
Or actually don't bother much — I will push changes in the next hour or so anyway.